cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -45,6 +45,10 @@
   // for backwards compatibility
   private final List<ETLStage> actions;
   private final Boolean service;
+  // support for SQL Engines
+  private final Boolean transformationPushdown;
```

Comment:
rather than having multiple properties at the top level, I think it would be better if it's just grouped into a single class.

Reply:
I will create a separate class for this.